LaTeX/TeX.sublime-syntax

```diff
@@ -116,15 +116,42 @@ contexts:
       scope: comment.line.percentage.tex
 
   controls:
-    - match: ((\\)(else|expandafter|fi|if(case|cat|dim|eof|false|hbox|hmode|inner|mmode|num|odd|true|undefined|vbox|vmode|void|x)?)){{endcs}}
+    - match: (\\)ifcase{{endcs}}
+      scope: keyword.control.conditional.switch.tex
       captures:
-        1: keyword.control.tex
-        2: punctuation.definition.backslash.tex
-    - match: ((\\)(?:input)){{endcs}}
-      scope: meta.function.input.tex
+        1: punctuation.definition.backslash.tex
+    - match: (\\)or{{endcs}}
+      scope: keyword.control.conditional.case.tex
```

Comment:
The `\or` looks more like an operator to me. Is there something like `\and` or `\not` as well?

```suggestion
      scope: keyword.operator.logical.tex
```

Reply:
I don't think it should be a logical operator. Then you'd expect \or to combine two boolean-like entities, right? I guess some languages (lua) idiomatically also use `or`-like constructs to get non-bool results, so that is not a strict criterion.

The following code from the tests 
```TeX
\ifcase 2 a\or b\or c\or d\else e\fi
```
would correspond more-or-less (I'd have to look up if TeX uses 0 or 1 to start counting) to the following `C` code:
```c
switch(2) {
 case 0: a; break;
 case 1: b; break;
 case 2: c; break;
 case 3: d; break;
 default: e;
}
```

Reply:
The scope is ok then.